README.md

```diff
@@ -6,4 +6,11 @@ SoteriumOS is a highly experimental project; we currently do not provide builds
 SoteriumOS is a ChromiumOS-based distribution focused on being privacy-friendly to all users by removing all unnecessary network requests, removing all forms of telemetry or analytics, and making the operating system functional and usable without Google binaries (i.e. no Google account); without sacrificing the security model of ChromiumOS. 
 
 ## Contributions
-Unless you explicitly state otherwise, any contribution(s) intentionally submitted for inclusion in SoteriumOS by you will be tri-licensed under the following three licenses: [Unlicense](https://github.com/dantiodev/soteriumos/blob/main/licenses/LICENSE-UNLICENSE), [MIT License](https://github.com/dantiodev/soteriumos/blob/main/licenses/LICENSE-MIT), and [Apache License 2](https://github.com/dantiodev/soteriumos/blob/main/licenses/LICENSE-APACHE-2). Licensees may use this software under the terms of any of these licenses at their leisure.
+Unless you explicitly state otherwise, any contribution(s) intentionally submitted for inclusion in SoteriumOS by you will be tri-licensed under the following three licenses: [Unlicense](https://github.com/dantiodev/soteriumos/blob/main/licenses/LICENSE-UNLICENSE), [MIT License](https://github.com/dantiodev/soteriumos/blob/main/licenses/LICENSE-MIT), and [Apache License 2](https://github.com/dantiodev/soteriumos/blob/main/licenses/LICENSE-APACHE-2). Licensees may use this software under the terms of any of these licenses at their leisure. If you use other licenses that do not allow relicensing under these three licenses, your contribution will have to be rejected.
```

Comment:
> If you use other licenses that do not allow relicensing under these three licenses, your contribution will have to be rejected.

I don't think this needs to be stated. Realistically, who is going to actually submit a patch and demand that we use a different licensing scheme just for them? These types of notices are pretty standard and don't require any further elaboration imo.